### ISSUE TITLE
Horrible audio and video syncing/looping issues after 2.0.0 update

### ISSUE THREAD

**Issue by ghost** [jellyfin-archive/jellyfin-roku-legacy#1593]
## Software Versions

- Jellyfin Server Version:  10.9.0 - "unstable" Docker image
- Roku Client Version:  2.0.0

## Describe the bug
On practically every show/movie, there are horrible audio/video desyncing issues and even looping issues

## How To Reproduce
1. Go to any movie/show
2. Let it play for a bit
3. Experience audio getting out of sync with video, or audio/video just looping over and over
4. If that doesn't happen, try enabling subtitles (enabling subtitles has also been giving a lot more "Error during playback" popups)

## Expected behavior
Audio and video should be in sync and not loop

## Connection Information

- Is server local or remote? Local
- Is server connection HTTP or HTTPS? "HTTPS"/TLS

**Comment by 1hitsong** [comment]
We need more info so we can try to recreate the issue. I've been running this release code for months and have not seen any desync issues.

Video file details, server logs, etc.

Does it do the same on the stable server version?

**Comment by cewert** [comment]
We only really support the stable version of the server. Can you reproduce this using server version `10.8.13`?

Are you coming from client version `1.6.6`? Did the files work on 1.6.6?

**Comment by ghost** [comment]
Yes, everything was working fine audio/video-wise up until the 2.0.0 update. Files that worked fine prior do not work fine now, and it happens with literally any show/movie in any audio/video encoding, and happens with or without subtitles. Additionally, I do not experience this in any other Jellyfin client, only the Roku version.

**Comment by weirdofhermiston** [comment]
I'm on 10.8.10 and having the same issue, but only when I skip or fast forward in a file. It only happens on the Roku app. This never happened with the previous version.

**Comment by mateusfn98** [comment]
Signed!! Same problem here.

**Comment by 1hitsong** [comment]
Again, we can't do anything until y'all supply video details so we can attempt to recreate this issue.

Telling us "Go to any movie/show" isn't correct because I've been using this code for a while and have not seen this issue.

**Comment by weirdofhermiston** [comment]
Here's media info for a file with issues. 

Unsolved.Mysteries.S05E07.480p.WEB-DL.AAC2.0.H.264-PLAYREADY
Container: mkv
Size: 331 MB

Video
Title: 480p H264 SDR
Codec: H264
AVC: Yes
Profile: Main
Level: 30
Resolution: 640x480
Aspect ratio: 4:3
Interlaced: No
Framerate: 29.970032
Bitrate: 925 kbps
Bit depth: 8 bit
Video range: SDR
Video range type: SDR
Color space: bt709
Color transfer: bt709
Color primaries: bt709
Pixel format: yuv420p
Ref frames: 1
NAL: 4

Audio
Title: AAC - Stereo - Default
Codec: AAC
Profile: LC
Layout: stereo
Channels: 2 ch
Bitrate: 125 kbps
Sample rate: 48000 Hz
Default: Yes
Forced: No
External: No

Subtitle
Title: English [CC] - Undefined - SUBRIP
Codec: SUBRIP
Default: No
Forced: No
External: No

**Comment by mateusfn98** [comment]
> Again, we can't do anything until y'all supply video details so we can attempt to recreate this issue.
> 
> Telling us "Go to any movie/show" isn't correct because I've been using this code for a while and have not seen this issue.

Open some media, change subtitles, backward skip to the beginning, and watch it. (do some skip/backward skip if you don't get the error)
This is what I do in every media I watch (my auto subtitle stopped some weeks ago, but not only in Roku), and always get that error of looping/unsync.

**Comment by ghost** [comment]
I have now received several reports from people who use my server, that also updated to the Roku 2.0.0 client, that they are experiencing the audio/video desync/loop issues. Non-Roku users are not experiencing these issues.

**Comment by iffycodes** [comment]
I am seeing the issue as well. It only appears if I pause and resume the stream at some point. It is especially noticable at the end of a stream where you will see the program end and then jump back in to the middle.

**Comment by mateusfn98** [comment]
> I am seeing the issue as well. It only appears if I pause and resume the stream at some point. It is especially noticable at the end of a stream where you will see the program end and then jump back in to the middle.

or when re-opens some media, this "reverse" happens, showing the 5:00 scenes at 3:00.
continue not so often, but commonly.

**Comment by jessielw** [comment]
Can confirm this exists across multiple Roku devices. Roku Premieres/+, Roku Ultra, and a couple Roku TVs. Was able to reproduce it on this app and the [new version](https://github.com/jellyfin/jellyfin-roku) as well. 

I will open an issue on the new version
